sbin/send_email.py

```diff
@@ -190,12 +212,18 @@ def send_email():
     """Send queued emails."""
     db, metadata = setup_db()
     queued_emails = get_email_queue(db)
-    if len(queued_emails) == 0:
-        return
     mail_client = construct_mail_client()
 
     success_count = 0
 
+    if "'" in EMAIL_INTERNAL_DOMAIN:
+        return
```

Comment:
What's the purpose of this check? Attempting to block a SQL injection?

Reply:
That was the idea before I added the parameterized queries. Will remove this.